src/Event.tsx

```diff
@@ -43,22 +43,25 @@ class Event extends React.Component<EventProps, EventState> {
 
   async componentDidMount(): Promise<void> {
     const coreClient = getClient().core()
-    const eventData = await coreClient.events().get(this.props.id)
-    const projectId = eventData.projectID
-
+    const event = await coreClient.events().get(this.props.id)
+    const authzList = await coreClient
+      .projects()
+      .authz()
+      .roleAssignments()
+      .list(
+        event.projectID || "", // Project ID will never actually be undefined
+        {
+          principal: {
+            type: authz.PrincipalTypeUser,
+            id: await getUserID()
+          },
+          role: "PROJECT_USER"
+        }
+      )
     this.setState({
-      event: eventData
+      event: event,
+      userHasLogAccess: authzList.items.length > 0
```

Comment:
@krancour I believe over here it will give error when items are not there. From backend in the case of no items, there is no items field. So, items.length will give error that `accessing property length of undefined`. Kindly confirm this once.

Thanks.

Reply:
You're right. I almost checked for it, too, but then a little voice in the back of my head said, "The TS compiler would tell you if that could be undefined." The same voice was telling me the backend returns an empty list anyway -- also wrong.

Thanks for catching this.

It's fixed in #73